library_builders.sh

```diff
@@ -200,11 +239,29 @@ function build_hdf5 {
     file hdf5*.tar.gz
```

Comment:
above: 1.12.1 does not work with CMake for some buggy reason, but we can use it with autotools.